problems/coj_problems.go

```diff
@@ -10,14 +10,39 @@ import (
         "io/ioutil"
 )
 
-// generate_request_url generates the URL to make a GET Resquest.
+// COJJudgement is the struct to represent a COJ judment for a submition.
+type COJJudgement struct {
+    Id        int     `json:"id"`
+    Date      string  `json:"date"`
+    Username  string  `json:"user"`
+    ProblemId int     `json:"problem"`
+    Status    string  `json:"judgment"`
+    Language  string  `json:"lang"`
+}
+
+
+// COJJudgmentResponse is the struct to represent a collection of COJ Judgments.
+type COJJudgmentResponse struct {
+    Collection [] COJJudgement
+}
+
+
+// SubmitionsProblemInfo represents the important information of the submitions to a problem
+// The values for HasSolved && HasTried are "YES" && "NO"
```

Comment:
I know I used strings in the example, bu I think a boolean would be better, what do you think?

Reply:
Also, why a new struct for these two fields only?

Reply:
I thought in use a map instead of the struct. I decided to use a struct  in case we want to know more information in a future.

Reply:
Agree to use Bool here.